tests/utils.py

```diff
@@ -27,7 +27,13 @@
         ],
     ),
 ]
-
+PARAMS_MESSAGE_LIST  [
+        date(2020, 3, 27),
+        [
+            {"WiadomoscId": 56994, "FolderWiadomosci": "Odebrane"},
+            {"WiadomoscId": 51768, "FolderWiadomosci": "Odebrane"},
```

Comment:
You're asserting in tests for `"IdPrzedmiot"` and `"IdPracownik"`, but you don't have any of these here. But that's not a problem, 'cause I'll need to change it after all anyway to pass the repo Travis CI tests.